shell/platform/android/io/flutter/plugin/editing/ScribePlugin.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugin.editing;
+
+import static io.flutter.Build.API_LEVELS;
+
+import android.annotation.TargetApi;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import io.flutter.embedding.engine.systemchannels.ScribeChannel;
+
+/**
+ * {@link ScribePlugin} is the implementation of all functionality needed for handwriting stylus
+ * text input.
+ *
+ * <p>The plugin handles requests for scribe sent by the {@link
+ * io.flutter.embedding.engine.systemchannels.ScribeChannel}.
+ */
+public class ScribePlugin implements ScribeChannel.ScribeMethodHandler {
+
+  private final ScribeChannel mScribeChannel;
+  private final InputMethodManager mImm;
+  @NonNull private final View mView;
+
+  @TargetApi(API_LEVELS.API_34)
+  @RequiresApi(API_LEVELS.API_34)
+  public ScribePlugin(
```

Comment:
That's the FlutterView from when ScribePlugin is created in FlutterView.java:

```java
      scribePlugin =
          new ScribePlugin(
              this, textInputPlugin.getInputMethodManager(), this.flutterEngine.getScribeChannel());
```

I need it in order to call [InputMethodManager.startStylusHandwriting](https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#startStylusHandwriting(android.view.View)). Is there a better way to get ahold of the View at that time that's called maybe?

Reply:
I am not so confident as to block the pr but I suspect that you can't make that variable final. That there is not a 1 to 1 relationship between flutter view and plugin instantiation. 

I am specifically thinking of add to app and cached engine scenarios. 
https://docs.flutter.dev/add-to-app/android/add-flutter-screen#step-3-optional-use-a-cached-flutterengine

None of our existing tests would catch this because they would not execute this code path.

Reply:
(From another thread) I've made a view setter so it can be updated if needed.